### ISSUE TITLE
Non distinct UUID causes hosts not to show up

### ISSUE THREAD

**Issue by reestr** [openshift/assisted-installer#189]
If you use assisted-installer with a VM provider which does not randomize UUID as part of a VM clone activity, the hosts will all share the same UUID which the assisted installer interface does not expect or handle.

Instead of a list of hosts showing up on the interface, only one host entry will show and it will switch between all discovered hosts which share that UUID.  If distinct UUID is a requirement for AI, then the interface should recognize when hosts all have the same UUID and advise user of the issue.

<img width="1476" alt="Screenshot 2021-01-22 at 21 37 08" src="https://user-images.githubusercontent.com/4464624/105543498-222ebd80-5cfa-11eb-85b7-b5121850f437.png">

<img width="1109" alt="Screenshot 2021-01-22 at 21 37 33" src="https://user-images.githubusercontent.com/4464624/105543511-278c0800-5cfa-11eb-9638-38d949b412db.png">

Although I have not tested yet, I am assuming for Parallels the [--regenerate-src-uuid](https://download.parallels.com/desktop/v13/docs/en_US/Parallels%20Desktop%20Pro%20Edition%20Command-Line%20Reference/23457.htm) parameter should be used when cloning.

**Comment by ohadlevy** [comment]
@avishayt any thoughts? can / should we generate our own uuid vs relay on bios uuid? perhaps a combination of uuid and mac address?

**Comment by avishayt** [comment]
I agree, we can use a MAC address (maybe sort the MAC addresses alphabetically and use the first one, in case of multiple interfaces).
Note that this might break a small number of installations where an agent started with using its motherboard UUID, then rebooted into the agent again (e.g., wrong boot order) using its MAC address.

@ronniel1 FYI, need to track with JIRA
